README.md

```diff
@@ -2,27 +2,29 @@
 
 [![Integration tests](https://github.com/MobilityData/mobility-catalogs/actions/workflows/integration_tests.yml/badge.svg?branch=issue%2F343%2Fcatalogs-prototype)](https://github.com/MobilityData/mobility-catalogs/actions/workflows/integration_tests.yml) [![Unit tests](https://github.com/MobilityData/mobility-catalogs/actions/workflows/unit_tests.yml/badge.svg?branch=issue%2F343%2Fcatalogs-prototype)](https://github.com/MobilityData/mobility-catalogs/actions/workflows/unit_tests.yml) [![Export catalogs to CSV](https://github.com/MobilityData/mobility-catalogs/actions/workflows/export_to_csv.yml/badge.svg?branch=issue%2F343%2Fcatalogs-prototype)](https://github.com/MobilityData/mobility-catalogs/actions/workflows/export_to_csv.yml) [![Join the MobilityData chat](https://badgen.net/badge/slack/%20/green?icon=slack)](https://bit.ly/mobilitydata-slack)
 
-The Mobility Database Catalogs is a project that provides a list of open mobility data sources from across the world, and the code to filter and manipulate them. [You can learn more about the Mobility Database here](https://database.mobilitydata.org/).
+The Mobility Database Catalogs is a list of open mobility data sources from across the world. [You can learn more about the Mobility Database here](https://database.mobilitydata.org/).
+
+To search sources easily, you can download [the CSV spreadsheet](#the-spreadsheet). If you want to filter for specific types of sources, [you can learn how to here](#get-and-filter-sources).
```

Comment:
I believe the title anchor has been changed below, so we'd need to change the `#the-spreadsheet` for `#browsing-and-consuming-the-spreadsheet`

```suggestion
To search sources easily, you can download [the CSV spreadsheet](#browsing-and-consuming-the-spreadsheet). If you want to filter for specific types of sources, [you can learn how to here](#get-and-filter-sources).
```